alvr/server/cpp/platform/linux/EncodePipelineAMF.h

```diff
@@ -93,6 +93,7 @@ class EncodePipelineAMF : public EncodePipeline
     int m_renderHeight;
 
     bool m_hasQueryTimeout = false;
+    
```

Comment:
You can remove this newline.

Reply:
I've added it because it fits the format of the windows version better